lib/ejabberd/config/ejabberd_module.ex

```diff
@@ -0,0 +1,67 @@
+defmodule Ejabberd.Config.EjabberdModule do
+  @moduledoc """
+  Module representing a module block in the configuration file.
+  It offers functions for validation and for starting the modules.
+
+  Warning: The name is EjabberdModule to not collide with
+  the already existing Elixir.Module.
+  """
+
+  @type t :: %{module: atom, attrs: [Attr.t]}
+
+  defstruct [:module, :attrs]
+  alias Ejabberd.Config.{EjabberdModule, Attr, Validation}
```

Comment:
This line does not compile with Elixir 1.1.1.
https://travis-ci.org/processone/ejabberd/jobs/146668610#L1700

Is it possible to use another syntax to make the code portable across more version of Elixir ?